src/components/AppWrapper/App/index.tsx

```
@@ -176,6 +178,15 @@ export default class App extends Component<InterfaceApp> {
       this.chatterZDSession = zdSession;
     } else if (resetChat) {
       this.resetChat({ metaFields });
+    } else if (javascriptEvent && eventCallbacks) {
+        const specificCallback = eventCallbacks[javascriptEvent.event_name];
+        const genericCallback = eventCallbacks['*'];
+        if (specificCallback) {
+            specificCallback(javascriptEvent);
+        }
+        if (genericCallback) {
```

What are `genericCallback`s useful for?

---

I found while testing my embed callbacks that it was very helpful having a catchall handler. More specifically it was useful while testing to be able determine if the event names were what I actually expected them to be. After chatting with Alex about it, we decided it may also be similarly useful for other Embed users to be able to define a catch-all handler while testing their callbacks.